.travis.yml

```diff
@@ -30,10 +31,15 @@ install: >
   fi
 
 before_script: >
-  source zjs-env.sh && make update && source deps/zephyr/zephyr-env.sh
+  source zjs-env.sh $PARTSIZE && make update && source deps/zephyr/zephyr-env.sh
 
 script: >
-  if [ "$TARGET" == "default" ]; then
-    make V=1; else
-    make V=1 "${TARGET}";
+  if [ "$TARGET" == "zephyr" ]; then
+    make V=1
+    make V=1 "JS=samples/TrafficLight.js"
+    make V=1 "JS=samples/I2C.js"
```

Comment:
Testing each JS sample could be done in parallel on a separate VM, but is that overkill?